src/kOS/Suffixed/ActiveResourceValue.cs

```diff
@@ -0,0 +1,32 @@
+﻿using kOS.Safe.Encapsulation;
```

Comment:
We now have 3 different classes for dealing with resources. I had active and aggregate as one but there was almost no code sharing between them so i thought this made more sense.